### PR TITLE
feat(dashboard): to create, go to customize page with store data

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

as-is: To create, just click create button at create-page
to-be: To create, 
1. click create button at create-page 
2. then, save data to create at dashboard info store
3. route to customize-page
4. click save button at customize-page

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue [hunk 1]
```diff
@@ -195,7 +195,8 @@ const updateDashboardData = async () => {
         await SpaceRouter.router.push({
             name: DASHBOARDS_ROUTE.DETAIL._NAME,
             params: {
-                dashboardId: props.dashboardId,
+                // FIXME:: change dashboardId when creating dashboard
+                dashboardId: props?.dashboardId ?? '',
```
Comment (minseolee) [inline src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:199]: I'll fix it when making api create logic

File: src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue [hunk 2]
```diff
@@ -222,6 +223,19 @@ const handleChangeVariableOptions = (propertyName: string, selected: string|stri
...
+};
+
+onMounted(() => {
+    window.addEventListener('beforeunload', handleUnload);
+});
+
+onBeforeUnmount(() => {
+    window.removeEventListener('beforeunload', handleUnload);
+});
+
```
Comment (minseolee) [inline src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:238]: I've added same logic  from cost-dashboard-page

File: src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue [hunk 3]
```diff
@@ -222,6 +223,19 @@ const handleChangeVariableOptions = (propertyName: string, selected: string|stri
     variableState.variableData[propertyName] = selected;
 };
 
+// for preventing refresh
+const handleUnload = (event) => {
+    event.preventDefault(); event.returnValue = '';
+};
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:229]: praise: Learned something new! thanx :)

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**